src/main/java/org/mule/extensions/jms/internal/connection/provider/GenericConnectionProvider.java

```diff
@@ -33,6 +36,7 @@
  * @since 1.0
  */
 @Alias("generic")
+@ExternalLib(name = "JMS Client", description = "Client which lets communicate with a JMS broker", type = JAR)
```

Comment:
why a jar? This one could be anything. Actually, most common thing is for it to be a maven dep. I know of no broker which you can connect to with one single jar. It's always big ass maven deps